internal/kafka/internal/handlers/cloud_providers.go

```diff
@@ -78,6 +80,8 @@ func (h cloudProvidersHandler) ListCloudProviderRegions(w http.ResponseWriter, r
 						maxCapacityReached := true
 						kafka.InstanceType = instType
 						kafka.Region = cloudRegion.Id
+						size, _ := h.kafkaConfig.GetFirstAvailableSize(instType)
```

Comment:
why ignore the error here?

Reply:
in theory an error should never occur. not sure what to do if it did in this case, but I will try to add error handing here